docs/data/ampli/cli.md

```diff
@@ -204,16 +203,6 @@ Configure and update source runtime. Follow the prompts to select a specific pla
 ✔ Successfully configured source.
 ```
 
-```shell
-USAGE
-  $ ampli configure [--deprecated-runtimes]
```

Comment:
We should keep an example of `ampli configure` but remove the reference to `--deprecated-runtimes`.

Reply:
we have an example of `ampli configure` above.

Reply:
Oops I meant `USAGE`. I think it would be good to keep the same structure as the other commands even if it's somewhat redundant.

```shell
USAGE
  $ ampli configure
OPTIONS
   None
EXAMPLES
  $ ampli configure
```

Reply:
Change applied